modules/storage-configuration/src/main/java/org/apache/ignite/internal/storage/configurations/StoragesConfigurationSchema.java

```diff
@@ -31,7 +32,7 @@ public class StoragesConfigurationSchema {
     /**
      * Storage engines configuration.
      */
-    @NamedConfigValue
+    @ConfigValue
```

Comment:
Why have we changed this from list to a single value? I was expecting several engines in `StoragesConfiguration`

Reply:
Because this behaviour implemented not by the NameConfigValue, but by the ConfigurationExtension approach.

The named value can't be used here, because:
- We want to have only one instance on every engine, but with the named config we can create {"rocksdb1": {"engineType": "rocksdb"}, "rocksdb2": {"engineType": "rocksdb"}}
- We don't want to duplicate the engine name like this : {"rocksdb": {"engineType": "rocksdb"}}
Please take a look at the PersistPageMemoryStorageEngineExtensionConfigurationSchema for example.